src/modules/currency/wallet/currency-wallet-api.js

```diff
@@ -356,10 +356,30 @@ export function makeCurrencyWalletApi (
         ? mul(exchangeData.depositAmount, multiplierFrom)
         : nativeAmount
 
+      const hasDestTag = exchangeData.deposit.indexOf('?dt=') !== -1
+      let destTag
+      if (hasDestTag) {
+        const splitArray = exchangeData.deposit.split('?dt=')
+        exchangeData.deposit = splitArray[0]
+        destTag = splitArray[1]
+      }
+
       const spendTarget: EdgeSpendTarget = {
         nativeAmount: nativeAmountForSpend,
         publicAddress: exchangeData.deposit
       }
+      if (hasDestTag) {
+        spendTarget.otherParams = {
+          uniqueIndentifier: Number(destTag)
```

Comment:
This needs to be a string. The plugin turns this into a number itself

Reply:
nope, passing it as a string, the plug in throws an error.

Reply:
unless you updated the plugin since I wrote this piece